### PR TITLE
Feat(UI)/SRT-27 Add video background

### PR DESCRIPTION
**Pull request by maymax777**
- Add video background

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -18,21 +18,23 @@ export const FooterTemplate = ({
...
-            <ul>
-              {category.items.map((item, j) => (
-                <li key={`nav-category-item-${j}`}>
-                  <Link url={item.url}>{item.label}</Link>
-                </li>
-              ))}
-            </ul>
-          </CollapsibleList>
-        ))}
+        <div className="footer-categories">
```
Comment (FrozenIce0617) [inline src/components/Footer.js:21]: @maymax777 , this file is duplicated with PR [#356](https://github.com/keep-network/website/pull/356/files) .

File: src/components/Header.js [hunk 2]
```diff
@@ -118,7 +118,11 @@ export const HeaderTemplate = ({ navItems = [] }) => {
                 </a>
               </li>
               <li>
-                <a href="#" className="primary" rel="noopener noreferrer">
+                <a
+                  href="#"
+                  className="btn btn-primary nav-item"
+                  rel="noopener noreferrer"
+                >
```
Comment (FrozenIce0617) [inline src/components/Header.js:125]: @maymax777 , this line is also duplicated with PR [#356](https://github.com/keep-network/website/pull/356/files) .

File: src/css/button.scss [hunk 3]
```diff
@@ -5,3 +5,42 @@ a.button {
 a.button-primary {
   @extend .btn-primary;
 }
+
+.btn {
```
Comment (FrozenIce0617) [inline src/css/button.scss:9]: @maymax777 , this file is duplicated with PR [#356](https://github.com/keep-network/website/pull/356/files) .

**Review by FrozenIce0617 — CHANGES_REQUESTED** [review]

**Comment by maymax777** [comment]
This PR involves only the part for video background and doesn't need to work with Header and Footer. Duplicating was happened cause of branching off #356 and it will be fixed after merging with former branch.

**Review by FrozenIce0617 — APPROVED** [review]

**Review by micahjlucas — APPROVED** [review]

**Pull request merged**